airflow/kubernetes/kube_client.py

```diff
@@ -15,12 +15,48 @@
 # specific language governing permissions and limitations
 # under the License.
 """Client for kubernetes communication"""
+
+from typing import Optional
 from airflow.configuration import conf
 
 try:
     from kubernetes import config, client
     from kubernetes.client.rest import ApiException  # pylint: disable=unused-import
+    from kubernetes.client.api_client import ApiClient
+    from kubernetes.client import Configuration
+    from airflow.kubernetes.refresh_config import (  # pylint: disable=ungrouped-imports
```

Comment:
`isort`, which we have now added might try to re-order this, so you'll probably have to add `# isort:skip` here too.